travisci/check_patch.py

```diff
@@ -0,0 +1,18 @@
+from subprocess import Popen, PIPE, STDOUT
```

Comment:
Shebang: #!/usr/bin/python3 (to ensure it is run under python3 interpreter)

Reply:
Ahh, this should save me from adding python3 in "python3 travisci/check_patch.py", thanks!